### PR TITLE
Added warning to external player tooltip

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
---
Added warning to external player tooltip
---

**Pull Request Type**
- [x] Feature Implementation

**Related issue**
https://github.com/FreeTubeApp/FreeTube/issues/2373

**Description**
Add a warning to the already existing tooltip message.

**Testing (for code that is not small enough to be easily understandable)**
I checked if the text has changed

**Desktop (please complete the following information):**
 - OS: win10
 - OS Version: 21h2
 - FreeTube version: bleeding edge

### CODE REVIEW

File: static/locales/en-US.yaml [hunk 1]
```diff
@@ -731,7 +731,7 @@ Tooltips:
       to quickly return to the default playback rate (1x unless it has been changed in the settings).
   External Player Settings:
     External Player: Choosing an external player will display an icon, for opening the
-      video (playlist if supported) in the external player, on the thumbnail.
+      video (playlist if supported) in the external player, on the thumbnail. Warning, Invidious settings does not affect external players.
```
Comment (absidue) [inline static/locales/en-US.yaml:734]:
```suggestion
      video (playlist if supported) in the external player, on the thumbnail. Warning, Invidious settings do not affect external players.
```

**Review by absidue — CHANGES_REQUESTED** [review]

**Review by absidue — APPROVED** [review]
Looks good to me, thanks.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> Looks good to me, thanks.

@absidue tested this in my ubuntu vm (always double check my pr's in there) and i didnt see the message change :(

**Comment by absidue** [comment]
@efb4f5ff-1298-471a-8973-3d47447115dc did you explicitly set the language in the FreeTube settings to en-US?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
> @efb4f5ff-1298-471a-8973-3d47447115dc did you explicitly set the language in the FreeTube settings to en-US?

yeah nvm i just wanted to comment that im stupid 😅

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
i had it set to `system default` and assumed it was the same as en-US

**Review by PikachuEXE — APPROVED** [review]

**Review by Svallinn — APPROVED** [review]

**Pull request merged**